develop-docs/development/ngrok.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Developing with Ngrok
+description: This document explains how to use ngrok with the development server.
+sidebar_title: Ngrok
+sidebar_order: 30
+---
+
+# Using ngrok
+
+The development server can be operated behind ngrok for when you need to receive traffic from the outside world. This comes up frequently when working with integrations. 
+
+## Ngrok in `monolith` mode
+
+The following are the steps will setup ngrok for integrations in `monolith` mode:
```

Comment:
The wording on this is a bit funny. Also, I'm not sure we need to mention `monolith` mode until we update the docs with silo'd ngrok instructions.